### PR TITLE
[LFX 2866] Feature/Redirect After Signing Wait

### PR DESCRIPTION
**Pull request by wanyaland**
- Added await for users checking cla-manager role for signed for company

Signed-off-by: wanyaland <wanyaland@gmail.com>

### CODE REVIEW

File: cla-backend/cla/controllers/signing.py [hunk 1]
```diff
@@ -207,6 +211,30 @@ def return_url(signature_id, event=None):  # pylint: disable=unused-argument
...
+            if signature.get_signature_reference_type() == 'company':
+                cla.log.info('Getting company instance : %s ', signature.get_signature_reference_id())
+                try:
+                    company = get_company_instance()
+                    company.load(str(signature.get_signature_reference_id()))
+                except DoesNotExist as err:
+                    cla.log.error('Invalid company_id provided : err: %s', signature.get_signature_reference_id)
+                user_service = UserService
+                cla.log.info('Checking if cla managers have cla-manager role permission')
+                while True:
```
Comment (dealako) [inline cla-backend/cla/controllers/signing.py:231]: Should we add a delay with a maximum number of attempts?

**Review by dealako — CHANGES_REQUESTED** [review]

File: cla-backend/cla/controllers/signing.py (hunk 1 above)
Comment (dealako) [reply cla-backend/cla/controllers/signing.py:231]: Sleep 0.5 seconds, retry up to N times?

Comment (wanyaland) [reply cla-backend/cla/controllers/signing.py:231]: So what would be the preferred number ?

**Review by dealako — APPROVED** [review]
👍

**Pull request merged**